spec/factories/data_types.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :data_type do
+    formula "MyString"
+name "MyString"
+  end
```

Comment:
Fix this indent for name "MyString", and change MyString to something that actually makes sense as a Formula and Name